### PR TITLE
Keyboards/RGBKB/Mün

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

RGBKB's Mun keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rgbkb/mun/mcuconf.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#pragma once
```
Comment (drashna) [inline keyboards/rgbkb/mun/mcuconf.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/mun.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#include "mun.h"
```
Comment (drashna) [inline keyboards/rgbkb/mun/mun.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/mun.h [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#pragma once
```
Comment (drashna) [inline keyboards/rgbkb/mun/mun.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/rev1/config.h [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (drashna) [inline keyboards/rgbkb/mun/rev1/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/mun.h [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+
+#include "rev1.h"
```
Comment (drashna) [inline keyboards/rgbkb/mun/mun.h:3]:
```suggestion
#if defined(KEYBOARD_rgbkb_mun_rev1)
#    include "rev1.h"
#endif
```

File: keyboards/rgbkb/mun/rev1/rev1.c [hunk 6]
```diff
@@ -0,0 +1,113 @@
+#include "rev1.h"
```
Comment (drashna) [inline keyboards/rgbkb/mun/rev1/rev1.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/rev1/rev1.h [hunk 7]
```diff
@@ -0,0 +1,34 @@
+#pragma once
```
Comment (drashna) [inline keyboards/rgbkb/mun/rev1/rev1.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/rgbkb/mun/rev1/rgbkb_mun.json [hunk 8]
```diff
@@ -0,0 +1,338 @@
+{
```
Comment (drashna) [inline keyboards/rgbkb/mun/rev1/rgbkb_mun.json:1]: Is a via json, please remove.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/rgbkb/common/touch_encoder.c [hunk 9]
```diff
@@ -0,0 +1,318 @@
...
+        xprintf("read_register %d failed %d\n", address, status);
+        return false;
+    }
+    return true;
+}
+
+void touch_encoder_init(void) {
+    i2c_init();
+
+    touch_handness = is_keyboard_left() ? 0 : 1;
```
[the remark this answers is not in the document]
Comment (XScorpion2) [reply keyboards/rgbkb/common/touch_encoder.c:155]: Believe it or not, that is actually bad practice when you need to consume the int value itself. Boolean is defined as 0 = false, any other value is true. So if you get in the habit of doing that, you could end up in a situation where your expected result is wrong. And yes, this has happened to me.

**Review by drashna — APPROVED** [review]

**Comment by Legonut** [comment]
Approved by RGBKB

**Review by kageurufu — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Review by fauxpark — COMMENTED** [review]
This PR has a few checklist problems and should not have been merged in its current state.

**Comment by XScorpion2** [comment]
> 
> 
> This PR has a few checklist problems and should not have been merged in its current state.

I'll reread the checklist and go fix them...been a while since I read that and it's probably changed.

**Pull request merged**